lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -4,11 +4,127 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-@use '../../scss-shared/nx-form-validation-helpers';
+@use '../../scss-shared/nx-container-helpers';
+@use '../../scss-shared/nx-validation-helpers';
 
 .nx-text-input {
```

Comment:
`NxTextInput` styles had previously been split up across three different files for historical reasons - it made some sense back when the component was simple enough to use via raw styles, and that was supported.  But that is no longer supported (hasn't been for a long time) and I found the necessary changes for this PR to be much easier to make after consolidating the styles.